homeassistant/components/sonos/services.yaml

```diff
@@ -35,87 +38,88 @@ snapshot:
   description: Take a snapshot of the media player.
   fields:
     entity_id:
-      description: Name(s) of entities that will be snapshot.
+      name: Entity
+      description: Name of entity that will be snapshot.
       example: "media_player.living_room_sonos"
       selector:
         entity:
           integration: sonos
           domain: media_player
     with_group:
-      description: True (default) or False. Also snapshot the group layout.
+      name: With group
+      description: True or False. Also snapshot the group layout.
       example: "true"
+      default: true
+      selector:
+        boolean:
 
 restore:
   name: Restore
   description: Restore a snapshot of the media player.
   fields:
     entity_id:
-      description: Name(s) of entities that will be restored.
+      name: Entity
+      description: Name of entity that will be restored.
       example: "media_player.living_room_sonos"
       selector:
         entity:
           integration: sonos
           domain: media_player
     with_group:
-      description: True (default) or False. Also restore the group layout.
+      name: With group
+      description: True or False. Also restore the group layout.
       example: "true"
+      default: true
+      selector:
+        boolean:
 
 set_sleep_timer:
   name: Set timer
   description: Set a Sonos timer.
+  target:
+    device:
+      integration: sonos
   fields:
-    entity_id:
-      description: Name(s) of entities that will have a timer set.
-      example: "media_player.living_room_sonos"
-      selector:
-        entity:
-          integration: sonos
-          domain: media_player
     sleep_time:
+      name: Sleep Time
       description: Number of seconds to set the timer.
       example: "900"
       selector:
         number:
           min: 0
-          max: 3600
-          step: 1
+          max: 7200
+          step: 60
```

Comment:
```suggestion
          step: 1
```

The UI will actually prevent submitting a value of `200` if you change the step size.

<img width="1172" alt="Screen Shot 2021-04-25 at 8 58 06 AM" src="https://user-images.githubusercontent.com/663432/116005828-5f7a4400-a5a4-11eb-9fb0-61b7055c3674.png">

Reply:
This is a sleep timer so figured people would not really care to use the timer in fractions of minutes

Reply:
I have an automation that uses fractions of a minute.